src/cloudburst/providers/aws/rds_instance.py

```diff
@@ -0,0 +1,151 @@
+"""
+RDS Resource for AWS provider
+"""
+
+from cloudburst.providers.base import AWSService, opcode
+from cloudburst.utils.resource_factory import aws_factory
+from cloudburst.utils.errors import NoResourcesError
+from cloudburst.utils.utils import aws_paginator
+from cloudburst.utils.shared_vars import AWS_REGIONS
+
+import boto3
+
+
+class RDSInstance(AWSService):
```

Comment:
Can you add a test case to the tests directory to cover this AWSService?

Reply:
Sure, I can do that. I've taken a look at test_ec2_instance.py, I'm happy to begin building out a similar test file for RDSInstance.